wagtailio/features/blocks.py

```diff
@@ -39,9 +39,37 @@ class Meta:
         template = "features/blocks/additional_features_block.html"
 
 
+class FeatureBlock(blocks.StructBlock):
+    heading = blocks.CharBlock(label="Feature heading", required=False, max_length=255)
+    description = blocks.RichTextBlock(
+        label="Feature description",
+        required=True,
+        features=["bold", "italic", "link", "document"],
+    )
+    link = blocks.URLBlock(label="Feature link", required=False)
+
+    class Meta:
+        icon = "gem"
+        label = "Feature"
+        template = "patterns/components/streamfields/features/feature_block.html"
+
+
+class FeatureCategoryBlock(blocks.StructBlock):
+    heading = blocks.CharBlock(required=False, max_length=255)
+    features = blocks.ListBlock(FeatureBlock(), min_num=2)
+
+    class Meta:
+        icon = "folder-inverse"
+        label = "Feature Category"
+        template = (
+            "patterns/components/streamfields/features/feature_category_block.html"
```

Comment:
Suggestion: I think it might be cleaner to use one template, and not provide templates to the child templates, instead of creating multiple nested-child templates. What do you think?

Reply:
Thanks @William-Blackie 

Are you referring to 

1. dropping the template definition for the **FeatureBlock**?

https://github.com/wagtail/wagtail.org/blob/7b09edd312419bd3bf96efea71d7033a2fb78b87/wagtailio/features/blocks.py#L54

  and instead having the markup defined in `patterns/components/streamfields/features/feature_category_block.html` ?
 
2. or  perhaps modifying `patterns/components/streamfields/features/feature_category_block.html` as follows:

```diff
<div>
    <h3>{{ value.heading }}</h3>
    {% for feature in value.features %}
-        {% include "patterns/components/streamfields/features/feature_block.html" with value=feature %}
+        {% include_block feature with value=feature %}
    {% endfor %}
</div>
```

Reply:
I've used the second approach in 81669e0, but happy to update if needed.